Zend/zend_signal.h

```diff
@@ -12,101 +12,53 @@
   | obtain it through the world-wide-web, please send a note to          |
   | license@php.net so we can mail you a copy immediately.               |
   +----------------------------------------------------------------------+
-  | Authors: Lucas Nealan <lucas@php.net>                                |
-  |          Arnaud Le Blanc <lbarnaud@php.net>                          |
-  +----------------------------------------------------------------------+
```

Comment:
I wouldn't remove this :)

Reply:
That's fine. Do you think the Zend signal handling authors will still want their names on a completely different source file which happens to have the same name, though?

Literally nothing has remained -- the only lines which are the same are the header comment and a couple of preprocessor directives like `#include <signal.h>`.

Reply:
Agree, doesn't make sense to retain authors here.